### ISSUE TITLE
exporting credentials from SSO authenticated sessions as ENV variables

### ISSUE THREAD

**Issue by brickheadz40625** [aws/aws-toolkit-jetbrains#3650]
please provide a possibility to export credentials as ENV vars from the tool to be used in shells NOT opened by the tool directly

---
Toolkit: AWS Toolkit For JetBrains 1.67-231
OS: Linux 5.19.0-41-generic
IDE: IntelliJ IDEA Ultimate Edition 231

**Comment by justinmk3** [comment]
Related feature for vscode toolkit (specifically the `Aws: Inject Credentials` setting): https://github.com/aws/aws-toolkit-vscode/pull/3040

**Comment by rli** [comment]
This is what the "local terminal" button does in the AWS Toolkit tool window

![image](https://user-images.githubusercontent.com/742829/235981629-7113f455-2b55-4014-a415-38e109284be0.png)

**Comment by rli** [comment]
actually it looks like this isn't working correctly in 231

**Comment by brickheadz40625** [comment]
please note that none of the env vars in the screenshot I attached are actually set as env vars in your output

.. also ..

you don't want/can use the terminal attached to the tool (i.e. use the button you are referring to)

tl;dr please make the funcitonality of the webpag (see screenshot) also available using the plugin (for generating AWS_ACCESS_KEY_ID, AWS_SECRET_ACCESS_KEY, AWS_SESSION_TOKEN)

**Comment by rli** [comment]
The AWS SDKs are capable of resolving credentials directly from the profile defined in `AWS_PROFILE`. Is there a specific reason why you need credentials in the form of the access key/secret key/token?

**Comment by brickheadz40625** [comment]
yes

a) it is not used by terraform properly even in the terminal opened from the tool https://www.google.com/search?q=terraform+s3+invalid+credentials 
b) there might be other edge cases where the AWS_ACCESS_KEY_ID, AWS_SECRET_ACCESS_KEY, AWS_SESSION_TOKEN env vars are needed

please note that the website provides a way to generate these so _if_ I need them (and sometimes I do) I generate them there

it would be more convenient though to be able to do it via the plugin (which might be one of the reasons the plugin exists at all I guess?)
